test/liquid_test_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# This can be used to setup for running tests from the liquid test suite.
+# For example, you could run a single liquid test as follows:
+# $ ruby -r./test/liquid_test_helper `bundle info liquid --path`/test/integration/template_test.rb
```

Comment:
TIL! I guess this can't be a rake task or such in this repo because it needs to run the bundler context of Liquid?

Reply:
There is already a rake task for running the liquid integration tests, however, that is for running all integration tests rather than specific ones.